build.gradle

```diff
@@ -22,7 +22,7 @@ repositories {
 }
 
 group 'bio.terra'
-version '0.0.22-SNAPSHOT'
+version '0.0.25-SNAPSHOT'
```

Comment:
nit: this is optional. GHA can figure this out without this change. (it reads https://github.com/DataBiosphere/terra-common-lib/tags and bump based on that)

Reply:
Ack – it was just helpful in my local testing, to make sure my local Maven publishing used the right version.

Reply:
Is it possible to use a local version string to make sure it doesn't get checked in? Like `0.0.1-LOCAL`?

Reply:
In a past project we used something like this (not sure it was this exact plugin), such that only major/minor changes were changed in source, and patch bumps happened implicitly: https://plugins.gradle.org/plugin/eu.davidea.grabver

Reply:
Thanks John – I added this option to the [turpentine log](https://docs.google.com/document/d/1JTQnS0FmuboW31wmmGMcjWww7wXix8ODbqcvkbQbtp4/edit#) doc.